marketplace/frontify-assets/extension.json

```diff
@@ -0,0 +1,28 @@
+{
+  "id": "frontify-assets",
+  "name": "Frontify Assets",
+  "srcdoc": "./build/index.html",
+  "fieldTypes": ["Object"],
+  "parameters": {
+    "installation": [
+      {
+        "id": "domain",
+        "type": "Symbol",
+        "name": "Frontify Domain",
+        "description": "Your Frontify domain, e.g. https://weare.frontify.com",
+        "default": "https://weare.frontify.com",
+        "required": true
+      }
+    ],
+    "instance": [
```

Comment:
I would make this parameter an installation parameter so it can be provided in the configuration screen of the app rather than in a field configuration.

Reply:
@jelz Frontify works a little different than other DAMs. Frontify offers distinct libraries for different types of Assets, eg. Media Library, Document Library, Template Library, Icon Library, Logo Library, etc. Moreover, clients can have multiple instances of each library type. 

Access tokens can then be created in a very flexible way, eg. access to a single library, access to multiple libraries and guidelines etc. Therefore I think our clients need the flexibility to define the access token per field. 

However, what would be cool at a later stage, is to have a "default token" as an installation parameter that can be overridden in the field configuration. Is this possible?

Reply:
Hey @brunschgi it makes sense. Could you please push your changes and then I'll add the logic you've described (default token)